packages/flutter/test/material/expansion_tile_theme_test.dart

```diff
@@ -47,8 +47,8 @@ class TestTextState extends State<TestText> {
 }
 
 void main() {
-  Material getMaterial(WidgetTester tester) {
```

Comment:
Would be good if we can add a unit test to show the new property in expansion tile theme can update the children's background color if widget.childrenBackgroundColor is null.